.dockerignore

```diff
@@ -1 +1,2 @@
 **/node_modules
+**/.git
```

Comment:
Why is this necessary out of curiosity?

Reply:
I could just do .git. There is no need to copy the git files to the docker container. The intent is to make for a smaller container.

Reply:
The "**/" will ignore the .git folder in any subdirectory.